src/client/components/Title/TitleDesc.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TitleDesc.styles.css'
+
+const TitleDesc = ({title,description}) =>{ return (<div> <h3 className="title">{title}</h3>
+<p className="description">{description}</p></div> ); };
+
```

Comment:
The indentation is a bit weird here

Reply:
?

Reply:
the code is compressed. use a line for every tag 
```suggestion
const TitleDescription = ({title,description}) =>{
 return (
 <div> 
 <h3 className="title">{title}</h3>
<p className="description">{description}</p>
</div> )
 };

```